client/sites-dashboard/components/sites-plan-renew-nag.tsx

```diff
@@ -24,9 +24,11 @@ const PlanRenewContainer = styled.div( {
 
 const PlanRenewLink = styled.a( {
 	whiteSpace: 'nowrap',
-	textDecoration: 'underline',
+	textDecoration: 'underline !important',
 	fontSize: '12px',
+	fontWeight: '400 !important',
 	paddingTop: '2px',
+	color: '#3858e9 !important',
```

Comment:
It seems like the DataViews component might be too opinionated with colors, maybe there's a fix to do in Gutenberg here as well.

Reply:
That would be great 🙏